common/org.testeditor.dsl.common.ui.tests/src/org/testeditor/dsl/common/ui/wizards/TestProjectConfigurationWizardPageTest.xtend

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.dsl.common.ui.wizards
+
+import org.junit.Test
+import org.mockito.Mockito
+import org.eclipse.swt.widgets.List
+
+/**
+ * Logic tests for TestProjectConfigurationWizardPage.
+ */
+class TestProjectConfigurationWizardPageTest {
+	
+	/**
+	 * Tests the Selection Listener to move the content from one list to the other list.
+	 */
+	@Test
+	def testUpdateListWithMoveListener() {
```

Comment:
public methods, especially those annotated with `@Test` should have an explicit return type. If the last line of the test is changed to a method that returns something, the inferred type will change and the test fail (JUnit complains if the return type is not `void`).